<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 219, 173);
border-color: rgb(176, 121, 70);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(21, 22, 24); 
font: 8pt &quot;Sitka Heading&quot;; font-size:36pt;</string>
   </property>
   <property name="text">
    <string>SiBook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_but">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>280</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
   </property>
   <property name="text">
    <string>Begin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
